2Questão/CodeReview.js

```diff
@@ -0,0 +1,67 @@
+// UserManagement.js
+import React from 'react';
+
+class UserManagement extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [
+        { id: 1, name: 'Alice', email: 'alice@example.com' },
+        { id: 2, name: 'Bob', email: 'bob@example.com' }
+      ],
+      newUserName: '',
+      newUserEmail: ''
+    };
+  }
+
+  handleNameChange(event) {
+    this.state.newUserName = event.target.value;
```

Comment:
```suggestion
    this.setState({ newUserName: event.target.value });
```
Utilize o setState para garantir que funcione corretamente.

Reply:
Utilize o setState para garantir que funcione corretamente.